developer/installation/installation-windows.md

```diff
@@ -0,0 +1,104 @@
+# Installation for Windows
+
+## Install prerequisites:
+
+**Note: These commands all assume you are running them from an administrator shell**
+
+### Install Node
+Download and run the installer from the [NodeJs site](https://nodejs.org)
+
+### Install Chocolately
+
+Visit the [Chocolatey site](https://chocolatey.org/install) and follow the instructions
+
+
+### Install Git
+
+```sh
+choco install git
+````
+
+### Install Meteor
+```sh
+choco install meteor
+```
+
+
+### Install Windows Build Tools 2015
+
+```sh
+npm install -g windows-build-tools
+```
+
+### Install ImageMagick (Optional but recommended)
+
+```sh
+choco install imagemagick
```

Comment:
Annotation 1: Isn't graphicsmagick the tool of choice, because faster?

Annotation 2: I think the part with imagemagick is missing in OSX installation guide.

Reply:
Do you have some source to cite for the part about graphicsmagick?

Reply:
Have heard that. Don't know if it's true. But at least the linux part of the installation is telling to install graphicsmagick. and I've seen references to graphicsmagick in Reaction source code somewhere.

Reply:
see: https://docs.reactioncommerce.com/reaction-docs/brent-fix-issue-401/installation-linux

Reply:
But don't the current OSX and Windows docs say to install ImageMagick?

Reply:
interesting..

Reply:
[scratches beard]